frontend/selectors/index.js

```diff
@@ -20,7 +20,7 @@ export const getShopifyVariant = createSelector(
   getProductById,
   (product) => {
     const { productData } = product || {};
```

Comment:
I think you should create a default value for productData here like 
```
const { productData = {} } = product || {};
```
This was done in the line that was removed  like `const { customData } = productData || {};`

I think it would makes sense to add a comment that explains why there is a productData.backInStockCustomData

Reply:
Updated and incorporated feedback from CCP-1678 PR.